samples/DP_SQL_dataverse_dataset.ipynb

```diff
@@ -24,8 +24,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from burdock.query.sql import PandasReader, execute_private_query\n",
-    "from burdock.metadata.collection import Collection, Table, Float"
+    "from burdock.sql import PandasReader, execute_private_query\n",
+    "from burdock.metadata.collection import CollectionMetadata, Table, Float"
```

Comment:
burdock.metadata import Colle*** ?

Reply:
CollectionMetadata is also in __init__ for burdock.sql.  I don't think we should add all of the other ones, since there would be name collisions, so assumption is that we import burdock.metadata.collection * only when doing literals.